ztrouy/gondor-backend
# Feature - Primary Address for Personal Account Details

### Purpose
To provide the Primary Address of the logged in User when fetching their personal account details.

### Files Changed
- Added related_name of "active_address" to the User property in `user_address.py`
- Added the `primary_address` calculated property that returns the Users' Address set to be their primary Address, if on exists in `user.py`
- Updated get_my_details method to include the Users' primary Address when possible in `user.py`

### To Test
Fetch, run the following commands in the project root directory
```
pipenv shell
```
```
pipenv install
```
```
./seed_database.sh
```
Then confirm the following
- GET endpoint `/users/me` returns the account details of the logged in User, including primary Address is exists
